src/converters/converter-pl.js

```diff
@@ -69,26 +104,28 @@ function generateWords(number) {
 	let num;
 	try {
 		num = parsedAndValid(number).toString();
-	}
-	catch (error) {
+	} catch (error) {
 		throw error;
 	}
 
 	amount = num.length;
 
 	for (i = 0; i <= amount; i++) {
 		tmp = num[i] - '0';
-		if (tmp > 9)
-			break;
+		if (tmp > 9) break;
 		digits[amount - i - 1] = tmp;
 	}
 
 	if (amount === 1 && digits[0] === 0) {
 		words = 'zero';
-	}
-	else {
+	} else {
 		for (i = parseInt((amount - 1) / 3); i >= 0; i--) {
```

Comment:
why `parseInt`?

Reply:
Cause we need only total part of this calculation. Without `parseInt` if f.e amount would be `1234.5` result would be `411.16`, so it would iterate 412 times instead of 411.

Reply:
It's needed in cases where e.g `amount - 1 === 2`, we want `i` to be an integer, because we use `i` as an index to access elements from `AMOUNTS` array